src/Personal/reducer.js

```diff
@@ -0,0 +1,67 @@
+import { actionType } from './actionCreator';
+
+const initialBackup = {
+  response: 'Click button to backup data',
+};
+
+const initialUser = {
+  username: '',
+  email: '',
+  accessToken: '0xa143981f3ec758296a1575146eab03ef047b7e40',
+};
+
+const backupState = (state = initialBackup, action) => {
+  switch (action.type) {
+    case actionType.BACKUP_START:
+      return {
+        ...state,
+        response: 'Backing up',
+      };
+    case actionType.BACKUP_DATA:
+      return {
+        ...state,
+        response: 'Back up successfully!',
+      };
+    case actionType.BACKUP_FAILED:
+      return {
+        ...state,
+        response: 'Ops, something went wrong',
+      };
+    case actionType.BACKUP_INITIAL:
+      return {
+        ...state,
+        response: 'Click button to backup data',
+      };
+    default:
+      return state;
+  }
+};
+
+const user = (state = initialUser, action) => {
```

Comment:
please remove this reducer, it is not related to this story

Reply:
I created it with Wen together, as the username and email display in UI needs the value from the user information store

Reply:
ok :)